template/module/__openerp__.py

```diff
@@ -19,7 +19,17 @@
         "base",
     ],
     "data": [
+        "security/some_model.xml",
+        "security/ir.model.access.csv",
+        "views/assets.xml",
+        "views/res_partner.xml",
+        "report/name.xml",
+        "wizard/wizard_model.xml",
     ],
     "demo": [
+        "demo/res_partner.xml",
```

Comment:
Same reasoning as before to distinguish in the file name the purpose of the xml without seeing the path.